openmetadata-service/src/test/java/org/openmetadata/service/resources/EntityResourceTest.java

```diff
@@ -810,7 +812,7 @@ void get_entityIncludeDeleted_200(TestInfo test) throws IOException {
   ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
   @Test
   @Execution(ExecutionMode.CONCURRENT)
-  protected void post_entityCreateWithInvalidName_400() {
+  protected void post_entityCreateWithInvalidName_400() throws IOException {
```

Comment:
Do sneakyThrows at the place where this method is overridden instead of adding this to the base method inherited by everyone.